tests/graphql_empty_file/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,11 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`empty-file.graphql 1`] = `
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Let's keep these tests in `tests/empty/`.

Just edit `tests/empty/jsfmt.spec.js` to add `graphql` and `json` to the parsers